File: src/1295-Find-Numbers-with-Even-Number-of-Digits/find_numbers_with_even_number_digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
  even_digits_count = 0
        
  for (let i = 0; i < nums.length; i++) {
    count = 0
    while (nums[i] >= 1) {
      nums[i] = nums[i] / 10
      count += 1
    }
    if (count % 2 == 0) {
      even_digits_count += 1
    }
  }
    
  return even_digits_count
};

nums = [12,345,2,6,7896]
console.log(findNumbers(nums))

File: src/1295-Find-Numbers-with-Even-Number-of-Digits/find_numbers_with_even_number_digits.py
from typing import List

class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        even_digits_count: int = 0
        
        for i in range(len(nums)):
            count: int = 0
            while (nums[i] >= 1):
                nums[i] = nums[i] / 10
                count += 1
            if count % 2 == 0:
                even_digits_count += 1
        
        return even_digits_count


if __name__ == "__main__":
    nums: List[int] = [12,345,2,6,7896]
    solution = Solution()

    print(solution.findNumbers(nums))